configure.ac

```diff
@@ -184,6 +184,15 @@ AC_CONFIG_FILES([Makefile
 		 src/tests/atlocal
 		 src/icons/Makefile])
 
+if test "x$ENABLE_DOCS" = "xyes"; then
+AC_CONFIG_COMMANDS([xsl-cleanup],,[rm -f doc/xml/transform-*.xsl])
+AC_CONFIG_FILES([doc/Makefile
```

Comment:
As per the [autoconf manual](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/autoconf.html#Configuration-Actions), it is indeed legit to call `AC_CONFIG_FILES()` multiple times:

```
This section describes [...] AC_CONFIG_FILES [...]
All these macros can be called multiple times, [...]
```